crates/next-custom-transforms/src/transforms/react_server_components.rs

```diff
@@ -281,6 +288,13 @@ fn report_error(app_dir: &Option<PathBuf>, filepath: &str, error_kind: RSCErrorK
                 .to_string(),
             vec![span],
         ),
+        RSCErrorKind::NextRscErrServerDirective(span) => (
+            "You're using the \"use server\" directive. That only works in a Server Component \
```

Comment:
```suggestion
            "You're using the \"use server\" directive. That only works in app router \
```
It also works in client components inline so maybe we don't specifically call out server component.